testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/StatisticsFetcher.java

```diff
@@ -90,30 +97,29 @@ private Stats toStats(Statistics statistics, Stats previousStats)
         }
 
         Stats stats = new Stats();
-        stats.systemCpuUsage = statistics.getCpuStats().getSystemCpuUsage();
-        stats.totalCpuUsage = statistics.getCpuStats().getCpuUsage().getTotalUsage();
+        stats.systemCpuUsage = firstNonNull(statistics.getCpuStats().getSystemCpuUsage(), -1L);
+        stats.totalCpuUsage = Optional.ofNullable(statistics.getCpuStats().getCpuUsage()).map(CpuUsageConfig::getTotalUsage).orElse(-1L);
```

Comment:
or 0 ?

Reply:
-1 so the cpu usage won't be calculated if there are no stats